docs/interpreter_status.md

```diff
@@ -0,0 +1,216 @@
+# StableHLO Interpreter
+
+The main goal of the StableHLO interpreter is to provide a reference
+implementation to the semantics of StableHLO opset according to its
+specification. The secondary goal is for the implementation to closely follow
+the spec to provide additional clarity to the semantics of even the most
+involved operations like `Convolution`, `Gather`/`Scatter`, and `DotGeneral`.
+
+At the moment, OpenXLA supports the interpretation of 91 out of 96 specced
+StableHLO ops, with the remaining 5 ops (`FftOp`, `RngOp`, `RngBitGeneratorOp`,
+`UniformDequantizeOp`, `UniformQuantizeOp`) being a work in progress (see
+[status.md](https://github.com/openxla/stablehlo/blob/main/docs/status.md) for
+a complete list of ops and its latest status).
+
+## Scope
+
+We categorized the StableHLO opset into 11 categories consisting of 118 ops in
+total (see [Appendix](#appendix)). According to our
+[roadmap](https://github.com/openxla/stablehlo/blob/main/docs/roadmap.md), one
+goal of StableHLO v1.0 is to implement a reference interpreter for all specced
```

Comment:
The actual wording is "We are planning to complete all or almost all work in this workstream in StableHLO v1.0."

Perhaps use the phrasing "all or most all specced ops" here to match this

Reply:
Good point, instead of reinventing the wording, I directly copied the wording for consistency.